src/main/java/first/dto/StudentDto.java

```diff
@@ -1,71 +1,58 @@
 package first.dto;
 
-public class StudentDto {
-    private String name;
-    private String surname;
-    private Integer age;
-    private Integer mark;
-    private Integer course;
-    private String button;
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public String getSurname() {
-        return surname;
-    }
+import first.model.Student;
+import first.repository.StudentRepository;
+import java.util.List;
 
+public class StudentDto implements StudDto, StudDtoShow {
+    private final StudentRepository rep = new StudentRepository();
 
-    public void setSurname(String surname) {
-        this.surname = surname;
+    public String add() {
+        return FORM;
     }
 
-    public Integer getAge() {
-        return age;
+    public String edit(Integer id) {
+        rep.getConnection();
+        return String.format( editForm, id, rep.getStudentById( id ).getName(), rep.getStudentById( id ).getSurname(), rep.getStudentById( id ).getAge(), rep.getStudentById( id ).getMark(), rep.getStudentById( id ).getCourse() );
     }
 
-    public void setAge(Integer age) {
-        this.age = age;
-    }
-
-    public Integer getMark() {
-        return mark;
-    }
+    public String editDone(String name, String surname, Integer age, Integer mark, Integer course, Integer id, String button) {
+        System.out.println( "Данные зашли" );
+        Student studentRead = new Student( id, name, surname, age, mark, course );
+        System.out.println( "Студент создан" );
+        StudentRepository stud = new StudentRepository();
+        stud.getConnection();
+        System.out.println( "подключились к базе" );
+        stud.update( studentRead );
 
-    public void setMark(Integer mark) {
-        this.mark = mark;
+        return strEdit;
     }
 
-    public Integer getCourse() {
-        return course;
+    public String del(Integer id) {
+        rep.getConnection();
+        return String.format( delForm, id, rep.getStudentById( id ).getName(), rep.getStudentById( id ).getSurname(), rep.getStudentById( id ).getAge(), rep.getStudentById( id ).getMark(), rep.getStudentById( id ).getCourse() );
     }
 
-    public void setCourse(Integer course) {
-        this.course = course;
+    public String delDone(Integer id) {
```

Comment:
I think, this is not good name because delete action has not already done. So it should be deleteStudentById